src/directive.js

```diff
@@ -14,6 +14,9 @@ export default {
   inserted: (el) => {
     el = core.getInputElement(el)
     const config = el[CONFIG_KEY]
+
+    if (typeof config.config.oldValue === 'undefined') core.updateValue(el, null, { force: config.prefill })
```

Comment:
_info_: If the input doesn't have a value when `bind` above is called, it won't establish the `oldValue`. If that's the case, try to update again here, in case the input value has been set subsequently (ex: if the value is set on component mounted, for example, [the element-ui input](https://github.com/ElemeFE/element/blob/dev/packages/input/src/input.vue#L431)).